android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -75,7 +77,7 @@ protected void onResume() {
         int spanCount = (int) (width/250.00);
 
         sessionRecyclerView.setHasFixedSize(true);
-        final GridLayoutManager gridLayoutManager = new GridLayoutManager(this,spanCount);
+        gridLayoutManager = new GridLayoutManager(this,spanCount);
```

Comment:
`gridLayoutManager = new GridLayoutManager(this,spanCount);`
Reformat this line in all the files.